tools/Dockerfile_test

```diff
@@ -0,0 +1,10 @@
+FROM fedora:35
```

Comment:
not f36?

Reply:
I wanted to have go1.16 but I can upgrade. No problem